dataplicity/m2m/wsclient.py

```diff
@@ -355,7 +355,6 @@ def run(self):
         sockopt = [(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)]
         try:
             self.app.run_forever(
```

Comment:
The heartbeat thread in ws4py was poorly implemented anyway, and no longer required.